src/lib/format/l20n/ast/parser.js

```diff
@@ -5,25 +5,32 @@ import { L10nError } from '../../../errors';
 
 const MAX_PLACEABLES = 100;
 
-export default {
-  parse: function(env, string, pos = false) {
-    this._source = string;
-    this._index = 0;
-    this._length = string.length;
-    this._curEntryStart = 0;
+
+class ParseContext {
+  constructor(env, pos) {
     this._config = {
       pos: pos
```

Comment:
Would it make sense to flatten this into `this._configPos` since there's only one config item right now?